Gruntfile.js

```diff
@@ -192,21 +192,21 @@ module.exports = function(grunt) {
      */
     banner:
       '/*!\n' +
-      ' *              ad$$             $$\n' +
-      ' *             d$"               $$\n' +
-      ' *             $$                $$\n' +
-      ' *   ,adPYba,  $$$$$ $b,dPYba,   $$,dPYba,\n' +
-      ' *  aP\'    \'$: $$    $$P\'   \'$a  $$P\'   \'$a\n' +
-      ' *  $(         $$    $$(     )$  $$(     )$\n' +
-      ' *  "b,    ,$: $$    $$b,   ,$"  $$b,   ,$"\n' +
-      ' *   `"Ybd$"\'  $$    $$`YbdP"\'   $$`Ybd$"\'\n' +
-      ' *                   $$\n' +
-      ' *                   $$\n' +
-      ' *                   ""\n' +
+      ' *               ad$$               $$\n' +
+      ' *              d$"                 $$\n' +
+      ' *              $$                  $$\n' +
+      ' *   ,adPYba.   $$$$$  $$.,dPYba.   $$.,dPYba.\n' +
+      ' *  aP′    `$:  $$     $$P′    `$a  $$P′    `$a\n' +
+      ' *  $(          $$     $$(      )$  $$(      )$\n' +
+      ' *  "b.    ,$:  $$     $$b.    ,$"  $$b.    ,$"\n' +
+      ' *   `"Ybd$"′   $$     $$`"YbdP"′   $$`"YbdP"′\n' +
+      ' *                     $$\n' +
+      ' *                     $$\n' +
+      ' *                     $$\n' +
       ' *\n' +
       ' *  <%= pkg.name %> - v<%= pkg.version %>\n' +
-      ' *  <%= pkg.homepage %>' +
-      ' *  A public domain work of the Consumer Financial Protection Bureau\n' +
+      ' *  <%= pkg.homepage %>\n' +
+      ' *  Licensed <%= pkg.license %> by <%= pkg.author.name %> <<%= pkg.author.email %>>\n' +
```

Comment:
This is going to output "Licensed https://github.com/cfpb/cfgov-refresh/blob/master/TERMS.md" if you don't update the value on `bower.json`.

I know this line comes from generator-cf, but I say leave it as-is for now. I'm not satisfied with the output of this for our projects using the default answers. The phrase "Licensed CC0 by ..." has a number of legal and grammatical issues.

I have more thoughts on this that don't really belong in this comment.